src/sentry/search/snuba/backend.py

```diff
@@ -64,6 +71,18 @@ def first_release_all_environments_filter(version, projects):
     )
 
 
+def inbox_filter(inbox, projects):
+    organization_id = projects[0].organization_id
```

Comment:
We don't necessarily need this, but it doesn't hurt much either.

Reply:
I'll probably take it out in a future PR